src/main/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/FagsakService.kt

```diff
@@ -60,6 +72,29 @@ class FagsakService(
         return assosierteFagsakDeltagere
     }
 
+    @Transactional
+    fun hentEllerOpprettFagsak(fagsakRequest: FagsakRequestDto): MinimalFagsakResponsDto {
+        val personident = when {
+            fagsakRequest.personIdent !== null -> fagsakRequest.personIdent
+            fagsakRequest.aktørId !== null -> fagsakRequest.aktørId
+            else -> throw Feil(
+                "Hverken aktørid eller personident er satt på fagsak-requesten. Klarer ikke opprette eller hente fagsak.",
+                "Fagsak er forsøkt opprettet uten ident. Dette er en systemfeil, vennligst ta kontakt med systemansvarlig.",
+                HttpStatus.BAD_REQUEST
+            )
+        }
```

Comment:
```suggestion
val personident = fagsakRequest.personIdent ?: fagsakRequest.aktørId ?: throw Feil(
"Hverken aktørid eller personident er satt på fagsak-requesten. Klarer ikke opprette eller hente fagsak.",
"Fagsak er forsøkt opprettet uten ident. Dette er en systemfeil, vennligst ta kontakt med systemansvarlig.", HttpStatus.BAD_REQUEST)
```

Hadde noe sånn her fungert? Er ikke så glad i nullchecker hvis vi kan unngå det med kotlin, men det er kanskje ikke like forståelig?

Reply:
ja, det kan sikkert fungere